workshop/Lab4/README.md

```diff
@@ -0,0 +1,74 @@
+# Lab 4 Deploy a service broker
+
+Learn to deploy a service broker, connect it to Kubernetes using Service Catalog,
+and view the catalog of services.
+
+# Open Service Broker API
+
+The [Open Service Broker API (OSBAPI)](https://www.openservicebrokerapi.org/) is an open
+standard that allows users of cloud platforms to provision, manage, and consume services
+in an automated fashion. We are going to create a service broker to automate deploying our
+Redis for us, as well as creating the secret that we consume in our deployment. If you're
+interested in what's going on behind the scenes, see [here](osbapi.md).
+
+# Deploy a broker
+
+For the purposes of this lab, we'll be using [Minibroker](https://github.com/osbkit/minibroker),
+a broker that lives inside our Kubernetes cluster. In the real world, the broker could be deployed
```

Comment:
`a broker that we will deploy to our Kubernetes cluster.`
Otherwise it sounds like its already deployed.